mesonbuild/compilers/compilers.py

```diff
@@ -877,6 +877,9 @@ def gnu_symbol_visibility_args(self, vistype: str) -> T.List[str]:
     def get_gui_app_args(self, value: bool) -> T.List[str]:
         return []
 
+    def get_win_subsystem_args(self, value: str) -> T.List[str]:
+        return []
```

Comment:
Wouldn't it make more sense to raise an `EnvironmentException` here? Since not setting the value is likely to lead to someone getting unexpected results?